### PR TITLE
Blogpost contribution about community discussion with focus on Nordics

### PR DESCRIPTION
**Pull request by naoe-tatara**
I would like to contribute a blog post about community discussion with focus on Nordic and Baltic region, held on the 30th October based on the participants' request. Hope this is not too late.

### CODE REVIEW

**Comment by kmomar** [comment]
Hi @naoe-tatara We can plan to publish this on Thursday, Dec 10 or early next week if that works for you.

File: _posts/2020/11/2020-11-24-Community-discussion-in-Nordic-and-Baltic-countries.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    - goal: making 3D holiday greeting card using HTML and JavaScript: https://arworkshop.teebusch.repl.co/card1.html
+    - The lesson material is all there:  https://repl.it/@Teebusch/arworkshop 
+- (Joakim) Suggestion of a repository of Carpentries Nordic/Baltic community members with specialities and skills
+- (Annika) Google Sheets for this?
+- (Anne) EOSC Nordic claims to be a knowledge-hub
+- (Radovan) Is also working on mapping.
+
+### Announcements
+
+- Nordic RSE (research software engineers) online get-together (Nov 30 - Dec 2): https://nordic-rse.org/events/2020-online-get-together/ (everybody welcome to attend and submit proposals or ideas)
```
Comment (naoe-tatara) [inline _posts/2020/11/2020-11-24-Community-discussion-in-Nordic-and-Baltic-countries.md]:
```suggestion
- Nordic RSE (research software engineers) online get-together (Nov 30 - Dec 2): https://nordic-rse.org/events/2020-online-get-together/ 
```

**Review by naoe-tatara — COMMENTED** [review]
Thanks, @kmomar ! Now the line 61 is a bit outdated, and it would be better to change a bit. Please take a look at the suggestion on changes below. If it look fine for the Carpentries, I will commit the suggestion.

File: _posts/2020/11/2020-11-24-Community-discussion-in-Nordic-and-Baltic-countries.md (hunk 1 above)
Comment (kmomar) [reply _posts/2020/11/2020-11-24-Community-discussion-in-Nordic-and-Baltic-countries.md]: Hi Naoe, that looks fine to me. Please go ahead and commit the change.

Comment (naoe-tatara) [reply _posts/2020/11/2020-11-24-Community-discussion-in-Nordic-and-Baltic-countries.md]: Done :)

**Comment by kmomar** [comment]
Great this will be published on 16 December!

**Pull request merged**